mindmap/mindmap.py

```diff
@@ -1,63 +1,72 @@
-"""XBlock to create Mind Maps in Open edX."""
+"""XBlock to create and save Mind Maps in Open edX."""
 
+import json
 import pkg_resources
 
+import boto3
+from django.conf import settings
 from django.template import Context, Template
 from django.utils import translation
+from web_fragments.fragment import Fragment
 from xblock.core import XBlock
-from xblock.fields import Integer, Scope, String
-from xblock.fragment import Fragment
+from xblock.fields import Scope, String
 from xblockutils.resources import ResourceLoader
 
 
+AWS_BUCKET_NAME = getattr(settings, "AWS_BUCKET_NAME", None)
```

Comment:
I don't know whether the settings you're loading are already available when this file is loaded. That's why I don't usually try getting settings here. Can we do it elsewhere? So we get it during the xblock runtime